# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../gimmage/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- gimmage-0.1.8/AppWindow.cpp.vanilla	2007-03-25 15:37:38.905771117 +0200
+++ gimmage-0.1.8/AppWindow.cpp	2007-03-25 15:37:43.706044669 +0200
@@ -19,7 +19,7 @@
 
 #include <gtkmm/stockid.h>
 #include <gdkmm/cursor.h>
-#include <gtk-2.0/gdk/gdkkeysyms.h>
+#include <gdk/gdkkeysyms.h>
 
 // we need this to unescape uris!
 #include <curl/curl.h>
